### PR TITLE
Add new extension

### PR DESCRIPTION
**Pull request by virus231**
## Description
Extension for web developers, quick documentation on almost every technology, which is related to web development.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by thomaspaulmann** [comment]
Can you please fill out the PR description? Thanks.

**Comment by virus231** [comment]
Add Description

**Comment by virus231** [comment]
> Can you please fill out the PR description? Thanks.
Add Description

File: extensions/frontend-cheatsheets/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Frontend Cheatsheets
+
+Help for developers
```
Comment (pernielsentikaer) [inline extensions/frontend-cheatsheets/README.md:3]: Could be cool with some more explanation here 🙂

File: extensions/frontend-cheatsheets/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "Web",
+    "Productivity",
+    "Documentation",
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Show List and Detail",
```
Comment (pernielsentikaer) [inline extensions/frontend-cheatsheets/package.json:19]: Did you forget to name your command?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

Is there a way to get the data in English? This extension looks quite handy but the language is going to be a huge problem for most of the users 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/frontend-cheatsheets/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "frontend-cheatsheets",
+  "title": "Frontend Cheatsheets",
+  "description": "Help for developers",
+  "icon": "command-icon.png",
```
Comment (pernielsentikaer) [inline extensions/frontend-cheatsheets/package.json:6]: You forgot to change icon (and delete the default)

**Pull request closed without merging**